plugins/inputs/diskio/diskio_linux_test.go

```diff
@@ -47,6 +48,7 @@ func TestDiskInfo(t *testing.T) {
 	require.NoError(t, err)
 	assert.Equal(t, "myval1", di["MY_PARAM_1"])
 	assert.Equal(t, "myval2", di["MY_PARAM_2"])
+	assert.Equal(t, "foo/bar/devlink foo/bar/devlink1", di["DEVLINKS"])
```

Comment:
Based on running udevadm we should prepend "/dev/" to the links: `/dev/foo/bar/devlink /dev/foo/bar/devlink1`.  I'm not sure if there are situations where the prefix would be something other than `/dev`, maybe in docker, but I think we could assume it for now unless we have a way to determine this.